js/main.js

```diff
@@ -1,2 +1,30 @@
 /* global data */
 /* exported data */
+
+var $url = document.querySelector('.photo-url');
+var $img = document.querySelector('.img');
+
+$url.addEventListener('input', function (event) {
+
+  $img.setAttribute('src', event.target.value);
+
+});
+
+var $form = document.querySelector('.form');
+
+$form.addEventListener('submit', function (event) {
+  event.preventDefault();
+  var formvalue = {
+    entryId: data.nextEntryId,
+    title: $form.elements.Title.value,
+    photourl: $form.elements.photoUrl.value,
+    notes: $form.elements.Note.value
+
+  };
+
+  data.nextEntryId++;
+  data.entries.push(formvalue);
+  localStorage.setItem('data-profile', JSON.stringify(data));
```

Comment:
make sure that all local storage functionality is completed in the data.ja file. Also, that functionality should match the functionality in the `javascript local storage` exercise. Local storage should only be used to load data at start, and store it at refresh.